backend/templates/403.html

```diff
@@ -0,0 +1,74 @@
+{% extends 'base.html' %}
+{% load static %}
+{% load i18n %}
+
+{% block title %}
+    {% blocktrans trimmed %}
+        403 Error
+    {% endblocktrans %}
+{% endblock %}
+
+{% block header %}
+    <script type="text/javascript">
```

Comment:
Remove, not needed

Reply:
I meant the whole block. There is no e-mail on that page, I'll commit it shortly

Reply:
@maltezacharias You can select multiple lines for a comment if you click and drag